Library/Homebrew/sorbet/rbi/gems/benchmark@0.4.0.rbi

```diff
@@ -0,0 +1,618 @@
+# typed: true
```

Comment:
Since sorbet already [includes](https://github.com/sorbet/sorbet/blob/master/rbi/stdlib/benchmark.rbi) this, we should exclude it in the tapioca config. (The sorbet version has typed annotations, so it's mostly the better one to use.)

Reply:
Updated the PR to avoid generating this. I also opened https://github.com/Homebrew/brew/pull/19075 to extend this approach to other stdlib gems (which this PR will conflict with).